src/routes/Search/Search.jsx

```diff
@@ -13,29 +14,44 @@ import Typography from '@material-ui/core/Typography';
 import SearchIcon from '@material-ui/icons/Search';
 
 import BusinessCard from '../../components/BusinessCard';
-import FilterDialog from '../../components/FilterDialog';
+import FilterPanel from '../../components/FilterPanel';
 import Pagination from '../../components/Pagination';
 
 import useSearch from './hooks/useSearch';
 import SearchForm from './SearchForm';
 
 const styles = (theme) => ({
   result: {
-    margin: '10px 0px 40px 0px',
+    // This is for setting the margins of the results returned from search.
```

Comment:
is this supposed to be empty ?

Reply:
At the time it was a placeholder comment in case I needed to adjust margins but I ended up going about it in a different way. It ends up being deleted in PR #81 but is still in PR #78 and #79.